statically linked libjpeg in libgs.so causes crash in evince
Bug # 15593209

NOTE : This patch will not be pushed upstream, as this only fix
       they way we want to link few libs to ghostscript in Solaris.

--- ghostscript-9.22/base/unix-gcc.mak.orig	2017-12-04 02:11:21.304884238 +0000
+++ ghostscript-9.22/base/unix-gcc.mak	2017-12-04 02:15:03.040553290 +0000
@@ -238,7 +238,7 @@
 # You may need to change this if the libpng version changes.
 # See png.mak for more information.
 
-SHARE_LIBPNG=0
+SHARE_LIBPNG=1
 PNGSRCDIR=./libpng
 LIBPNG_NAME=png
 
@@ -253,7 +253,7 @@
 # Define the directory where the zlib sources are stored.
 # See zlib.mak for more information.
 
-SHARE_ZLIB=0
+SHARE_ZLIB=1
 ZSRCDIR=./zlib
 #ZLIB_NAME=gz
 ZLIB_NAME=z
